### ISSUE TITLE
Enabling fix doesn't seem to update source files

### ISSUE THREAD

**Issue by duclet** [webpack-contrib/eslint-webpack-plugin#43]
- Operating System: Manjaro Linux
- Node Version: v13.9.0
- NPM Version: 6.13.7
- webpack Version: 5.4.0
- eslint-webpack-plugin Version: 2.3.0

### Expected Behavior

Running webpack with the fix option enabled should update the source code and autofix the rules that it can.

### Actual Behavior

If running without the fix option, it shows the error. If running with the fix option enabled, no errors are shown but source code is not updated.

### Code

```js
// webpack.config.js
...
   plugins: [
        new CleanWebpackPlugin({ verbose: true }),
        new VueLoaderPlugin(),
        new ESLintWebpackPlugin({ extensions: ['.ts', '.vue'], fix: true }),
        new PrettierPlugin({ extensions: ['.js', '.ts', '.vue'] }),
    ],
    module: {
        rules: [
            { test: /\.vue$/, loader: 'vue-loader' },
            {
                test: /\.ts$/,
                exclude: /node_modules/,
                loader: 'ts-loader',
                options: { appendTsSuffixTo: [/\.vue$/] },
            },
            { test: /\.js$/, loader: "source-map-loader" }
        ]
    }
...
```

```js
// .eslintrc
{
  "root": true,
  "parser": "vue-eslint-parser",
  "parserOptions": {
    "parser": "@typescript-eslint/parser",
    "sourceType": "module"
  },
  "plugins": ["simple-import-sort"],
  "rules": {
    "simple-import-sort/sort": "error"
  }
}

```

### How Do We Reproduce?

With configs above, running npx webpack in theory should fix the following file by re-ordering the imports:

```js
import MyInternalClass from './my-internal-class';
import BindAll from 'lodash-decorators/bindAll';
import shuffle from 'lodash/shuffle';
```

**Comment by starzou** [comment]
I have the same problem.

**Comment by starzou** [comment]
@ricardogobbosouza Can you take a look at this problem? Thank you.

**Comment by Liqiaomiao** [comment]
I have the same problem.
`configureWebpack: config => {
    config.devtool =
      config.mode === 'development'
        ? 'cheap-module-eval-source-map'
        : 'source-map';
    return  {
      plugins:[new ESLintWebpackPlugin({
        context:'src',
        extensions: ['.vue','.js'],
        fix: true,
      })]
    }
  },`

**Comment by rook-tx** [comment]
Same problem here, what's the latest?

Node 15.0.1
npm 7.0.3
webpack 4.44.2
eslint-webpack-plugin 2.4.0

**Comment by starzou** [comment]
@alexander-akait @jsg2021 

**Hello, It is still not fixed.**

**Comment by cave-** [comment]
Still occurring in 2.5.2. @jsg2021 
Anyone has the same problem? @starzou @Liqiaomiao @rook-tx
